mitodl/micromasters
# Added test for auto migrations (#2365)

#### What are the relevant tickets?

Fixes #2365 

#### What's this PR do?

This PR adds a test for migrations that didn't have a `--name` specified when they're created.

Also refactored detect_missing_migrations.sh to the script directory I created here and ensured ./test_suite.sh runs flow while I was in there.

#### Where should the reviewer start?

scripts/test/no_auto_migrations.sh

#### How should this be manually tested?

Run the script above, see that it prints nothing and exits `0`. Add a dummy auto migration file like `touch profiles/migrations/00300_auto_dummy_test_file.py` and run the script again. It fail print a message indicating the offending filename and exit nonzero.
